src/strategies/eth-wallet-age/index.ts

```diff
@@ -0,0 +1,45 @@
+import fetch from 'cross-fetch';
+
+export const author = 'chaituvr';
+export const version = '0.1.0';
+
+const delay = interval => new Promise(resolve => setTimeout(resolve, interval));
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+  ) {
+  const apiLimit = 300;
```

Comment:
Why you need this?

Reply:
Etherscan has an API limit - https://info.etherscan.com/api-return-errors/

```5 calls per sec/IP with a valid API key```

so 1000/5 ms should be the gap between each API call.. even if we give 200 sometimes it is exceeding the limit. works great with 300